runner-install.sh

```diff
@@ -8,4 +8,5 @@ sudo pip3 install pandas
 sudo apt-get install -y cmake
 sudo apt install -y unzip
 sudo apt-get -y install python3-boto3 python3-botocore # amazon.aws.aws_s3
```

Comment:
we need to install python3 as python

Reply:
added sudo apt install -y python-is-python3 as a convenient way to make symlink python to python3